config/root/clusterworkspace-home-get-clusterrole.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:kcp:home-clusterworkspace-get
+rules:
+- apiGroups: ["tenancy.kcp.dev"]
+  resources: ["clusterworkspaces/workspace"]
+  resourceNames: ["~"]
+  verbs: ["get"]
```

Comment:
just to be on the same page: this stays "get" only, correct? I.e. you cannot list?